contract_manager/scripts/latency_entropy_with_callback.ts

```diff
@@ -0,0 +1,86 @@
+import yargs from "yargs";
+import { hideBin } from "yargs/helpers";
+import { DefaultStore, toPrivateKey } from "../src";
+import { COMMON_DEPLOY_OPTIONS } from "./common";
+import Web3 from "web3";
+
+const parser = yargs(hideBin(process.argv))
+  .usage(
+    "Requests a random number from an entropy contract and measures the\n" +
+      "latency between request submission and fulfillment by the Fortuna keeper service.\n" +
+      "Usage: $0 --contract <entropy_contract_id> --private-key <private-key>"
+  )
+  .options({
+    contract: {
+      type: "string",
+      demandOption: true,
+      desc: "Contract to test latency for",
+    },
+    "private-key": COMMON_DEPLOY_OPTIONS["private-key"],
+  });
+
+async function main() {
```

Comment:
There are a lot of common parts between this script and `latency_entropy.ts`. Maybe you can make the request part a standalone function?

Reply:
So, the request part (userRandomNumber, providerInfo) in latency_entropy is being used in the reveal part too. 
For latency_with_callback this isn't the case. It is all closely linked

Reply:
hmm, what if we make a function that returns anything necessary for later?